src/common.rs

```diff
@@ -764,6 +765,32 @@ pub fn print_unsigned_transaction(transaction: &crate::commands::PrepopulatedTra
                 };
                 print_unsigned_transaction(&prepopulated_transaction);
             }
+            near_primitives::transaction::Action::DeployGlobalContract(deploy) => {
+                let code_hash = CryptoHash::hash_bytes(&deploy.code);
+                let identifier = match deploy.deploy_mode {
+                    GlobalContractDeployMode::CodeHash => {
+                        format!("deploy code <{:?}> as a global hash", code_hash)
+                    }
+                    GlobalContractDeployMode::AccountId => {
+                        format!(
+                            "deploy code <{:?}> to a global account <{}>",
+                            code_hash, transaction.receiver_id
+                        )
+                    }
+                };
+                eprintln!("{:>5} {:<70}", "--", identifier)
+            }
+            near_primitives::transaction::Action::UseGlobalContract(contract_identifier) => {
+                let identifier = match contract_identifier.contract_identifier {
+                    GlobalContractIdentifier::CodeHash(hash) => {
+                        format!("use <{}> code for deploy", hash)
+                    }
+                    GlobalContractIdentifier::AccountId(ref account_id) => {
+                        format!("use <{}> code for deploy", account_id)
```

Comment:
nit: `from` word can be added after `deploy`

as in
`use global <{}> code to deploy from`